superset/tasks/schedules.py

```diff
@@ -79,11 +86,13 @@ def _get_recipients(
 
 
 def _deliver_email(
-    schedule: Union[DashboardEmailSchedule, SliceEmailSchedule],
+    schedule: Union[DashboardEmailSchedule, SliceEmailSchedule, Alert],
     subject: str,
     email: EmailContent,
 ) -> None:
     for (to, bcc) in _get_recipients(schedule):
+        logging.info("Sending email to [%s] bcc [%s]", to, bcc)
```

Comment:
Let's not write email addresses to logs. This is protected information in most organizations.

Reply:
maybe do it debug logging? so it is configurable if needed ?